lib/cmd/fhc/target.js

```diff
@@ -106,7 +106,13 @@ function target(argv, cb) {
 
       var _gotVersion = function() {
         ini.save(function(err) {
-          target.message = i18n._("Successfully targeted: ") + targ.target + (targ.user ? (" User: " + targ.user) : (" API Key: " + targ[keys.KEY_ID]));
+          var message;
+          if (targ.user) {
+            message = " User: " + targ.user;
+          } else {
+            message = " API Key: " + targ[keys.KEY_ID];
+          }
+          target.message = i18n._("Successfully targeted: " + targ.target + message);
```

Comment:
I think @grdryn  is right here. The `User :` or `API Key: ` part is not in the `i18n._` before the change. Better keep it the same.